internal/crypto/crypto_test.go

```diff
@@ -0,0 +1,32 @@
+package crypto
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestHash(t *testing.T) {
+	testCases := []struct {
+		name string
+		salt string
+	}{
+		{
+			name: "without salt",
+			salt: "",
+		},
+		{
+			name: "with salt",
+			salt: "n pepa",
```

Comment:
👏

Reply:
btw... This is a good candidate to get moved to brigade-foundations in a future PR.